lib/node_modules/@stdlib/array/base/with/test/test.js

```diff
@@ -0,0 +1,163 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var Int32Array = require( '@stdlib/array/int32' );
+var Complex128Array = require( '@stdlib/array/complex128' );
+var toAccessorArray = require( '@stdlib/array/base/to-accessor-array' );
+var Complex128 = require( '@stdlib/complex/float64' );
+var updateatindex = require( '@stdlib/array/base/with/lib' );
```

Comment:
Please use the `with` alias instead of `updateatindex` in the test file as well.

```suggestion
var with = require( './../lib' );
```